src/components/Confirm/Confirm.js

```diff
@@ -30,6 +30,7 @@ const CALLBACK_TYPES = {
   video: 'onSubmitVideo',
   document: 'onSubmitDocument',
 }
+const UNEXPECTED_ERROR = 'REQUEST_ERROR'
```

Comment:
Minor: extra line break should be good here

```suggestion
const UNEXPECTED_ERROR = 'REQUEST_ERROR'

```